mmv1/products/compute/Router.yaml

```diff
@@ -150,7 +150,7 @@ properties:
           This enum field has the one valid value: ALL_SUBNETS
         item_type: Api::Type::String  # TODO(#324): enum?
         send_empty_value: true
-      - !ruby/object:Api::Type::Array
+      - !ruby/object:Api::Type::Set
         name: advertisedIpRanges
```

Comment:
This is currently causing a build error - to specify a field as a Set in these yaml files you will need to revert it back to  `!ruby/object:Api::Type::Array` and then add the property `is_set: true` to that field. However if you do that please refer to my previous review where making the change to a Set can only be in a major version.